### ISSUE TITLE
Cloner not working

### ISSUE THREAD

**Issue by ghost** [citadel-station/citadel-main#101]
Cloner has been reported as not working.
Apparently you can be scanned but not cloned.

**Comment by LetterJay** [comment]
I'm not sure how to reproduce this. 

I used two keys to test. Shot and killed one then cloned it. It worked normally. The only issue I saw was it giving a false "Failed to initialize" warning in the cloner console, but it still cloned.

**Comment by LetterJay** [comment]
Marking this as low priority. The only actual bug is the error message it sometimes gives when trying to clone. It actually clones, despite giving that message. Players also only go back into their bodies once the cloning is actually finished as opposed to it putting you in the moment you start being cloned.

**Comment by LetterJay** [comment]
Evidently people are now reporting they're not being cloned with their original sprites. IE: Be a wolf, get cloned into a default human.

Bumping this to normal priority.

**Comment by LetterJay** [comment]
```runtime error: 
[21:47:33]Cannot read null.prefs
[21:47:33]proc name: growclone (/obj/machinery/clonepod/proc/growclone)
[21:47:33]  source file: cloning.dm,189
[21:47:33]  usr: Riley Hayhurst (/mob/living/carbon/human)
[21:47:33]  src: the cloning pod (/obj/machinery/clonepod)
[21:47:33]  usr.loc: the floor (154,114,1) (/turf/open/floor/plasteel/white)
[21:47:33]  src.loc: the floor (155,113,1) (/turf/open/floor/plasteel/whiteblue/side)
[21:47:33]  call stack:
[21:47:33]the cloning pod (/obj/machinery/clonepod): growclone("jayehh", "Lucas Fleming", "5365362cc85f290542264e486c40c3...", "444365411042623313511550513544...", "\[0x2101055e]", Canid (/datum/species/canid), /list (/list), /list (/list))
[21:47:33]the cloning console (/obj/machinery/computer/cloning): Topic("src=\[0x20094e9];clone=8afe", /list (/list))
[21:47:33]TheLetterJay (/client): Topic("src=\[0x20094e9];clone=8afe", /list (/list), the cloning console (/obj/machinery/computer/cloning))```

**Comment by LetterJay** [comment]
Pointing to cloning file for the actual pod.

```	if(H.client.prefs)
		H.vore_organs = H.client.prefs.belly_prefs.Copy()
		for(var/I in H.vore_organs)
			var/datum/belly/B = H.vore_organs[I]
			B.owner = H
			B.internal_contents = list()
			B.digest_mode = DM_HOLD```

**Comment by LetterJay** [comment]
Hoping this was fixed in d27ec3b4fefc16c1b3daee4511d07cbcce54dc1e
